src/components/location/location.repository.ts

```diff
@@ -131,6 +149,7 @@ export class LocationRepository extends DtoRepository(Location) {
         ])
         .return<{ dto: UnsecuredDto<Location> }>(
           merge('props', {
+            mapImage: { id: 'props.mapImage' },
             fundingAccount: 'fundingAccount.id',
             defaultFieldRegion: 'defaultFieldRegion.id',
             defaultMarketingRegion: 'defaultMarketingRegion.id',
```

Comment:
Need to adjust these like `mapImage` to fit your DTO changes. That's why tests are failing.